### PR TITLE
343816 add button to delete connection on server show

### PR DESCRIPTION
**Pull request by B-Rass**

### CODE REVIEW

File: config/locales/fr.yml [hunk 1]
```diff
@@ -413,13 +416,18 @@ fr:
...
+      destroy_connections_confirmation: Confirmez-vous vouloir supprimer toutes les connections de ce matériel ?
     new:
       title: "Ajouter un matériel"
     edit:
       title: "Modifier le matériel"
       delete_confirmation: Toutes les données concernant ce matériel seront supprimées, ainsi que toutes les connexions associées. Souhaitez-vous confirmer ?
     duplicate:
       title: Dupliquer le matériel %{server_name}
+    flashes:
+        connections_destroyed: Les connexions ont été supprimées
```
Comment (B-Rass) [inline config/locales/fr.yml:429]:
```suggestion
        connections_destroyed: Les connexions ont été supprimées.
```

File: config/locales/fr.yml [hunk 2]
```diff
@@ -413,13 +416,18 @@ fr:
...
+        connections_not_destroyed: Une erreur a empêché la suppression des connexions
```
Comment (B-Rass) [inline config/locales/fr.yml:430]:
```suggestion
        connections_not_destroyed: Une erreur a empêché la suppression des connexions.
```

**Review by nicolas-brousse — CHANGES_REQUESTED** [review]

File: spec/requests/servers_request_spec.rb [hunk 3]
```diff
@@ -246,4 +246,12 @@
...
     end
   end
+
+  describe "GET /destroy_connections" do
+    before { get destroy_connections_server_path(server) }
+
+    it { expect(response).to have_http_status(:redirect) }
+    it { expect(server.connections.count).to be_zero }
+    it { expect(flash[:notice]).to be_present }
+  end
```
[the remark this answers is not in the document]
Comment (B-Rass) [reply spec/requests/servers_request_spec.rb:256]: Sure, but I can't find a case when it will fail

**Review by nicolas-brousse — APPROVED** [review]

File: spec/requests/servers_request_spec.rb (hunk 3 above)
[the remark this answers is not in the document]
Comment (nicolas-brousse) [reply spec/requests/servers_request_spec.rb:256]: I don't have any case in head. So let keep this like it is

**Pull request merged**